src/shared/codelens/typescriptCodeLensProvider.ts

```diff
@@ -107,10 +112,21 @@ export function initialize({
                 handlerName: params.handlerName,
                 templatePath: params.samTemplate.fsPath
             })
-            await invokeLambda({
-                runtime,
-                ...params,
-            })
+
+            if (params.isDebug && unsupportedNodeJsRuntimes.has(runtime)) {
+                vscode.window.showErrorMessage(
+                    localize(
+                        'AWS.lambda.debug.runtime.unsupported',
+                        'Debug support for {0} is currently not supported',
```

Comment:
On the fence on whether we should mention that `run` works...I can't really think of a message that doesn't sound clunky though.

Reply:
We're pretty clear about run vs debug elsewhere in the UX.